recipe/meta.yaml

```diff
@@ -6,133 +6,139 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
   skip: true  # [py!=27]
 
 requirements:
   build:
     - python
   run:
-    - alabaster
-    - appnope  # [osx]
-    - arb
-    - backports.shutil_get_terminal_size
-    - backports_abc
-    - bdw-gc
-    - boost-cpp
-    - brial
-    - cddlib
-    - certifi
-    - cliquer
-    - configparser
-    - cvxopt
-    - cycler
-    - cysignals
-    - decorator
-    - docutils
-    - ecl
-    - eclib
-    - ecm
-    - entrypoints
-    - fflas-ffpack
-    - flintqs
-    - fplll
-    - fpylll
-    - freetype
-    - functools32
-    - future
-    - gap
-    - gf2x
-    - gfan
-    - giac
-    - givaro
-    - glpk
-    - gmp
-    - gsl
-    - iml
-    - ipykernel
-    - ipython
-    - ipython_genutils
-    - ipywidgets
-    - jinja2
-    - jsonschema
-    - jupyter_client
-    - jupyter_core
-    - lcalc
-    - libflint
-    - libgap
-    - libgd
-    - libiconv
-    - libpng
-    - linbox
-    - lrcalc
-    - m4ri
-    - m4rie
-    - matplotlib
-    - maxima
-    - mistune
-    - mpc
-    - mpfi
-    - mpfr
-    - mpmath
-    - nauty
-    - nbconvert
-    - nbformat
-    - ncurses
-    - networkx
-    - notebook
-    - ntl
-    - numpy
-    - openblas
-    - palp
-    - pari
-    - path.py
-    - pathlib2
-    - pexpect
-    - pickleshare
-    - planarity
-    - ppl
-    - prompt_toolkit
-    - psutil
-    - ptyprocess
-    - pycrypto
-    - pygments
-    - pynac
-    - pyparsing
-    - python
-    - python-dateutil
-    - pytz
-    - pyzmq
-    - ratpoints
-    - readline
-    - rpy2
-    - rubiks
-    - rw
+    # We require all dependencies with at least that version that upstream uses
+    # in their SPKGs. For packages that (seem to) use semantic versioning and
+    # are at least 1.0.0, we limit the version to the next major release.
+    - alabaster >=0.7.8
+    - appnope >=0.1.0  # [osx]
+    - arb >=2.11.1,<3
+    - backports.shutil_get_terminal_size >=1.0.0,<2
+    - backports_abc >=0.5
+    - bdw-gc >=7.2f,<8
+    - boost-cpp >=1.61.0,<2
```

Comment:
@saraedum, what do you think about removing the pinnings for those that are already in `sagelib`? This looks weird.